api_service.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+class ApiService:
+	"""
+    This class will be responsible for communicating with our Heroku hosted API
+    to create and update existing desks
+    """
+	BASE_URL = "https://smart-library-app.herokuapp.com"
```

Comment:
Should we move this to config?

Reply:
Hmm I think we should always test against our production API. 

The good thing about VCR cassettes is that we can call the production API only once, and reuse the response saved in the cassette for subsequent test runs. You can find some more info here: http://www.qdonnellan.com/blog/2015/08/15/api-testing-with-vcrpy-why-have-i-not-done-this-before.html